doc/ug/install_instructions/index.md

```diff
@@ -55,6 +55,10 @@ For packages listed below without a version number we have not determined a mini
 * libusb 1.0 (called libusbx in older distributions)
 * OpenOCD 0.11.0
 * [Verible](https://github.com/google/verible) {{< tool_version "verible" >}}
+* Cmake 2.8.11
+* Perl 5.6.1
+* [Go](https://golang.org/) 1.10
+* [Rust](https://www.rust-lang.org/) 1.52.1
```

Comment:
I'd expect us to change the Rust version regularly; can you the version number into tool_requirements.py?

And then refer to it like this:

```suggestion
* [Rust](https://www.rust-lang.org/) {{< tool_version "rust" >}}
```

And this is currently only rust for x86_64? Maybe worth mentioning as well.